pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/Bytes.java

```diff
@@ -347,6 +347,25 @@ public int read() throws IOException {
         };
     }
 
+    /**
+     * Compare this {@link Bytes} object to another {@link Bytes} object. The comparison is done on a byte-by-byte
+     * @param otherData the object to be compared.
+     * @return a negative integer, zero, or a positive integer as this object is less than, equal to, or greater than
+     */
+    @Override
+    public int compareTo(Bytes otherData) {
+        // Compare the lengths first
+        final int minLength = Math.min(length, otherData.length);
+        for (int i = 0; i < minLength; i++) {
+            final int compare = Byte.compare(getByte(i), otherData.getByte(i));
```

Comment:
Should this be compareUnsigned?

Reply:
This probably wouldn't matter much as long as it's consistently applied, however `ByteString` class from Protobuf contains `ByteString#UNSIGNED_LEXICOGRAPHICAL_COMPARATOR` which suggests that unsigned comparison is the way to go. So, I'll do as you propose.